src/hc_sr04_driver/src/hc_sr04.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+# Always use a voltage divider to connect the HC SR04 sensor to the Raspberry Pi
+
+import RPi.GPIO as GPIO
+import time
+import rospy
+from std_msgs.msg import Float32
+
+# Set pinout mode of the Raspberry Pi
+GPIO.setmode(GPIO.BCM)
+TRIG_PIN = 23
+ECHO_PIN = 24
+SETTLE_TIME = 2
+ECHO_PULSE_TIME = 0.00001
+SPEED_OF_SOUND = 34300/2 # 0.5 speed of sound, as required by the formula
+PRECISION = 2
+QUEUE_SIZE = 10
+FREQ = 5
+
+# Driver function for the HC SR04 sensor
+def driver():
+    GPIO.setup(TRIG_PIN, GPIO.OUT)
+    GPIO.setup(ECHO_PIN, GPIO.IN)
+    GPIO.output(TRIG_PIN, False)
+    time.sleep(SETTLE_TIME)
+    pub = rospy.Publisher("distance_sensor", Float32, queue_size=QUEUE_SIZE)
+    rospy.init_node("hc_sr04", anonymous=True)
+
+    # Keep polling the sensor and push the data to the ROS network
+    rate = rospy.Rate(FREQ)
+    while not rospy.is_shutdown():
```

Comment:
Maybe in the future we could change this to not run permanently.
I don't think we need it to run all the time, only at select moments. 
For instance, when we turn to a can, we want to know what our distance to it is, we could call the function to run until we stop it when we reach the can.
Would the pi lose a lot of processing power over constantly doing this? I guess we should test this for a bit.